src/dotnet-razor-tooling/project.json

```diff
@@ -13,18 +13,20 @@
     "xmlDoc": true
   },
   "frameworks": {
-    "dnxcore50": {
+    "netstandard1.5": {
       "dependencies": {
         "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
         "Microsoft.AspNetCore.Razor.Runtime": "1.0.0-*",
         "Microsoft.DotNet.ProjectModel.Loader": "1.0.0-*",
         "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
-        "NETStandard.Library": "1.5.0-*",
-        "System.Diagnostics.Process": "4.1.0-*",
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-*"
```

Comment:
Was talking about this with @davidfowl today because we want to update all our tools, tests, and sites, to use shared framework. However, I'm not sure how we choose a version. @davidfowl ?